src-internal/Admin/Schedulers/CustomersScheduler.php

```diff
@@ -3,13 +3,16 @@
  * Customer syncing related functions and actions.
  */
 
-namespace Automattic\WooCommerce\Admin\Schedulers;
+namespace Automattic\WooCommerce\Internal\Admin\Schedulers;
 
 defined( 'ABSPATH' ) || exit;
 
 use \Automattic\WooCommerce\Admin\API\Reports\Cache as ReportsCache;
 use \Automattic\WooCommerce\Admin\API\Reports\Customers\DataStore as CustomersDataStore;
-use \Automattic\WooCommerce\Admin\Schedulers\OrdersScheduler;
+use Automattic\WooCommerce\Internal\Admin\Schedulers\ImportScheduler;
+use Automattic\WooCommerce\Internal\Admin\Schedulers\OrdersScheduler;
```

Comment:
@moon0326 I just found my PhpStorm shows lines 12-13 are unnecessary, though VSCode doesn't show this. I saw you also add additional imports to `OrdersScheduler.php` and `ImportScheduler.php`. Do we prefer these explicit imports?

![Screen Shot 2022-02-16 at 11 46 25](https://user-images.githubusercontent.com/4344253/154192457-85225f28-416e-4ceb-bdff-0bc6f44cc94a.png)

Reply:
Thank you for spotting them. They should be removed. I'm not sure why my PHPStorm added them (or did not remove them). I'll make an update.

Reply:
Fixed in https://github.com/woocommerce/woocommerce-admin/pull/8316/commits/149bd59853ba54715bbb8e796a2963e906e473dc